libs/domain/cart/list/list.component.ts

```diff
@@ -0,0 +1,41 @@
+import { CartService } from '@spryker-oryx/cart';
+import { resolve } from '@spryker-oryx/di';
+import { ContentMixin } from '@spryker-oryx/experience';
+import { HeadingTag } from '@spryker-oryx/ui/heading';
+import { signal } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { cartListStyles } from './list.styles';
+
+export class CartListComponent extends ContentMixin(LitElement) {
+  static styles = cartListStyles;
+  protected cartService = resolve(CartService);
+  protected $carts = signal(this.cartService.getCarts());
+
+  protected override render(): TemplateResult | void {
+    const carts = this.$carts();
+
+    return html`
```

Comment:
What about keep this component only focussing on the list and move out heading and text:
- CartListTitleComponent
- standard text component

Reply:
refactored